test/integration/grpc.go

```diff
@@ -48,14 +48,17 @@ func (t *grpc) run() error {
 }
 
 func (t *grpc) makeRequests() error {
-	testPods := []string{"a", "b"}
+	srcPods := []string{"a", "b"}
+	dstPods := []string{"a", "b"}
 	if t.Auth == proxyconfig.ProxyMeshConfig_NONE {
 		// t is not behind proxy, so it cannot talk in Istio auth.
-		testPods = append(testPods, "t")
+		srcPods = append(srcPods, "t")
```

Comment:
add "t" to dstPods too?

Reply:
or maybe just add "headless" to "testPods"?

Reply:
We can't use "headless" as a source (unless we explicitly remap it to the b pod).